pilot/pkg/security/authn/v1beta1/policy_applier.go

```diff
@@ -351,24 +348,42 @@ func GetMostSpecificScopeConfig(rootNamespace string, configs []*model.Config) *
 		if spec.Selector == nil || len(spec.Selector.MatchLabels) == 0 {
 			level += 2
 		}
-		if isStrictlyStronger(spec, configByScope[level]) {
-			configByScope[level] = cfg
-		}
+		policiesByScope[level] = append(policiesByScope[level], spec)
 	}
 
-	for _, cfg := range configByScope {
-		if cfg != nil {
-			return cfg
+	parentMtls := &v1beta1.PeerAuthentication_MutualTLS{
+		Mode: v1beta1.PeerAuthentication_MutualTLS_PERMISSIVE,
+	}
+	var candidatePolicy *v1beta1.PeerAuthentication
```

Comment:
you mean `consolidatedPolicy` rather than canddiate? is this a typo?

Reply:
outdated.